scripts/mysql_install_moodle.sh

```diff
@@ -625,14 +625,11 @@ EOF
     /usr/bin/unzip -q moodle.zip
     /bin/mv -v moodle-'$moodleVersion' /moodle/html/moodle
 
-    # Commented out as the plugin is not available
     # install Office 365 plugins
-    #if [ "$installOfficePlugins" = "True" ]; then
```

Comment:
Looks like this change will always install the O365 plugins, they should be optional. Not all Moodle users need them.

Reply:
Fair enough, but my excuse is that we already install the ElasticSearch plugin (and dependencies) and the 3 VMs without any option. Shouldn't we make all these optional then? Note that that'll be a bigger code change.

Reply:
Yeah - that should be optional too :-)